presto-parquet/src/main/java/com/facebook/presto/parquet/reader/ParquetReader.java

```diff
@@ -418,8 +440,10 @@ private ColumnChunkMetaData getColumnChunkMetaData(ColumnDescriptor columnDescri
             throws IOException
     {
         for (ColumnChunkMetaData metadata : currentBlockMetadata.getColumns()) {
-            if (metadata.getPath().equals(ColumnPath.get(columnDescriptor.getPath()))) {
-                return metadata;
+            if (!HiddenColumnChunkMetaData.isHiddenColumn(metadata)) {
+                if (metadata.getPath().equals(ColumnPath.get(columnDescriptor.getPath()))) {
```

Comment:
NIT: merge those two ifs to be more concise

Reply:
sure